Sources/App/Fastlane/SlackCommand+Fastlane.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import Vapor
+import Stevenson
+
+extension SlackCommand {
+    static let fastlane = { (ci: CircleCIService) in
+        SlackCommand(
+            name: "fastlane",
+            help: """
+            Invokes specified lane on specified branch (or develop if not specified).
+            Provide options the same way as when invoking lane locally.
+
+            Parameters:
+            - name of the lane to run
+            - list of lane options in fastlane format (e.g. `device:iPhone5s`)
+            - branch: name of the branch to run the lane on
+
+            Example:
+            `/fastlane test_babylon \(Option.branch):develop`
+            """,
+            allowedChannels: ["ios-build"],
+            run: { metadata, request in
```

Comment:
[nit] should we use trailing closure for those `SlackCommands` (both fastlane and CRP)?

Reply:
We could, but that's what you changed working on CRP command. I don't have preference.